gen/configs/aarch64-linux-gnu.toml

```diff
@@ -0,0 +1,16 @@
+[general]
+library_name = "libcurl"
+output_file_path = "../src/wrappers/aarch64-linux-gnu.jl"
+
+use_julia_native_enum_type = true
+use_deterministic_symbol = true
+auto_mutability = true
+
+printer_blacklist = [
+    "CURL_SUFFIX_CURL_OFF_T",
+    "CURL_SUFFIX_CURL_OFF_TU",
+    "CURL_ZERO_TERMINATED",
+]
+
+[codegen.macro]
+macro_mode = "basic"
```

Comment:
Since all of these config files are identical except for the `output_file_path` why not have a `common.toml` file which contains all of the common settings and allow for `$target.toml` files which can overwrite the defaults. Additionally, I would just remove `output_file_path` entirely here.

Reply:
I completely removed those toml files by programmatically creating the config option.

Reply:
The API for cross-platform support is still being designed(I need more feedback in real-world projects like libCURL).